### ISSUE TITLE
Feature Request: Add HPA Support

### ISSUE THREAD

**Issue by eduardobaitello** [mkimuram/k8sviz#23]
Congrats on this awesome project!

Would be nice to have [HPA (Horizontal Pod Autoscaler) ](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)support as well :)

Thanks!

**Comment by mkimuram** [comment]
@eduardobaitello 

Thank you for your feature request.
I've implemented it in #24 and k8sviz now show below figure for resources created by following commands:
```
$ kubectl apply -f https://k8s.io/examples/application/php-apache.yaml
$ kubectl autoscale deployment php-apache --cpu-percent=50 --min=1 --max=10
$ ./k8sviz.sh -n default -t png -o hpa.png
```

![hpa](https://user-images.githubusercontent.com/36450643/112352645-50e8f600-8ca1-11eb-91ff-f00fbed0204c.png)

Please check if it works as expected.

**Comment by mkimuram** [comment]
Closing this issue, for it should be resolved. If anything missing, please feel free to reopen.
